themes/default/content/blog/announcing-kubernetes-helm-release-resource-preview/index.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Announcing Preview for Helm Release Resource for Kubernetes
```

Comment:
```suggestion
title: Full access to Helm features with new Helm Release resource
```

Would love to encapsulate the value this new resource enables in the title, if possible. Help grab the user's attention.

Reply:
Fixed.